### PR TITLE
Huawei misc fixes

### PR DESCRIPTION
**Pull request by Dil3mm4**
The USB configuration stuff is tested. 
The QC properties are untested, but that was only a deletion. 
The persists are NOT tested.

### CODE REVIEW

File: base.mk [hunk 1]
```diff
@@ -24,12 +24,12 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
 	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-	ro.adb.secure=0
```
Comment (phhusson) [inline base.mk]: burp. please remove that from the commit

**Comment by phhusson** [comment]
vndk-detect is inside a non permisive selinux context, so I don't really know whether the setprops will work, so i wouldn't mind if you could test.

**Comment by Dil3mm4** [comment]
Last time we checked for QC stuff they weren't working only on "ro". 
But persists were doing their job. (also, you're already setting props for BT that way)

File: base.mk [hunk 2]
```diff
@@ -23,8 +23,7 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.codename=$(PLATFORM_VERSION_CODENAME) \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
-	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
```
Comment (phhusson) [inline base.mk]: that's really not what I meant...
I wanted you to **edit** your **commit** to remove the **change** you had there.
I don't want to remove ro.adb.secure, it's fine there.
But you had a change with various whitespaces
I don't want to squash your commits, I want them individual, hence I want you to really edit commits (and thus remove this commit)

**Comment by Dil3mm4** [comment]
So you want a pull request per each commit?

**Comment by phhusson** [comment]
No, I want one pull-request with three commits

**Comment by Dil3mm4** [comment]
Updates:
The persists are TESTED and WORKING.

**Pull request merged**